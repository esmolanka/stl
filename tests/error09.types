
type Foo (f : Â±Type -> Type) -a = f a
