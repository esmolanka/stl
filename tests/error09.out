tests/error09.stl:2:37-2:38: error:
    Contravariant variable 'a' must not be used in covariant positions.